lib/KLogo/index.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+
+  <div>
+    <KImg
+      :src="logo"
+      :altText="alternateText"
+      :style="styleObject"
+    />
+  </div>
+
+</template>
+
+<script>
+
+  import KImg from '../KImg.vue';
+  import kolibriLogo from './kolibri-logo.svg';
+
+  export default {
+    name: 'KLogo',
+    props: {
+      /**
+       * Alternate text for the image. This is required and will throw a warning when it's not provided (empty) unless isDecorative is true
+       */
+      altText: {
+        type: String,
+        default: '',
+      },
+      /**
+       * Sets the height for the component
+       */
+      height: {
+        type: [Number, String],
+        default: undefined,
+      },
+      /**
+       * Sets the width for the component
+       */
+      width: {
+        type: [Number, String],
+        default: undefined,
+      },
+      /**
+       * Sets the maximum height for the component
+       */
+      maxHeight: {
+        type: [Number, String],
+        default: undefined,
+      },
+      /**
+       * Sets the minimum height for the component
+       */
+      minHeight: {
+        type: [Number, String],
+        default: undefined,
+      },
+      /**
+       * Sets the maximum width for the component
+       */
+      maxWidth: {
+        type: [Number, String],
+        default: undefined,
+      },
+      /**
+       * 	Sets the minimum width for the component
+       */
+      minWidth: {
+        type: [Number, String],
+        default: undefined,
+      },
+    },
+    data() {
+      return {
+        styleObject: {
```

Comment:
`styleObject` and all related calculations are not needed here because `KImg` will take care of it after max/min/height/width/ props are sent there:

```
<KImg
      :src="logo"
      :altText="alternateText"
      :height="height"
      :width="width"
      ....
/>
```

Reply:
Done!!